lnwallet/interface_test.go

```diff
@@ -772,7 +772,7 @@ func testReservationInitiatorBalanceBelowDustCancel(miner *rpctest.Harness,
 		LocalFundingAmt:  fundingAmount,
 		RemoteFundingAmt: 0,
 		CommitFeePerKw:   feePerKw,
-		FundingFeePerKw:  feePerKw,
+		FundingFeePerKw:  1000,
```

Comment:
Why was this needed?

Reply:
The test is attempting to test that an insane commit fee is pushing our balance into the negative, but without this change it would fail even to select coins for the funding tx (this was not a problem before since we first selected coins after making the reservation).